tests/behat/features/templates/contact_us.feature

```diff
@@ -4,22 +4,24 @@
 
 Feature: test contact us form
 
-  Scenario: Unauthenticated user fills out the contact us form
+  Scenario: Unauthenticated user fills out the contact us form  
     Given I am not logged in
     When I am on the homepage
     When I follow "Contact Us"
-    And I should see "Your name"
-    And I should see "Your email address"
-    And I should see "Subject"
-    And I should see "Message"
-    # And I should see "Send message"
+    When I fill in "edit-name" with "Test"
+    When I fill in "edit-mail" with "Test@email.com"
+    When I fill in "edit-subject-0-value" with "Test"
+    When I fill in "edit-message-0-value" with "Test"
+    When I click "op"
+    #Recaptcha not working
```

Comment:
Please add a comment that op is the name of the submit button. 
Add a test after When I click "op":
    Then I should get a "200" HTTP response
Remove the comment about recaptcha. We disabled Recaptcha on dev environments.